content/en/platform/corda/4.12/enterprise/node/operating/tvu/running-tvu.md

```diff
@@ -0,0 +1,26 @@
+---
+description: "The steps that TVU goes through when verifying pre-4.12 database of transactions."
+date: '2024-11-04'
+menu:
+  corda-enterprise-4-12:
+    identifier: corda-enterprise-4-12-running-tvu
+    parent: corda-enterprise-4-12-tvu
+tags:
+- tvu
+- transaction validator utility
+title: Running the TVU
+weight: 150
+---
+
+# Running the TVU
+
+This section describes the requirements to run the Transaction Validator Utility (TVU) to verify a pre-4.12 database of transactions. Please read this section in conjunction with [Corda Enterprise Edition 4.11 to 4.12 upgrade guide]({{< relref "../../../upgrade-guide.md" >}}) which describes in more detail the upgrade process.
+
+The TVU is compatible only with 4.11 database schemas. To verify transactions on a pre-4.11 database, you must first upgrade the database to version 4.11. You must run the TVU in the same environment as a 4.12 node, in terms of `cordapps`, `legacy-contract`, and `legacy-jar` folders, except that the database it connects to is the 4.11 database being checked. Before performing the following steps, ensure everything is backed up, as changes will be made to folders as described below.
```

Comment:
Can you replace this paragraph with the following please,

The TVU is compatible only with 4.11 database schemas. To verify transactions on a pre-4.11 database, you must first upgrade the database to version 4.11. You must run the TVU in the same environment as a 4.12 node, in terms of `cordapps`, and `legacy-jar` folders, with the exception that the database it connects to is the 4.11 database being checked. Note that you do not need to create a legacy-contracts folder when running the TVU. The TVU will extract what it needs from the database. Before performing the following steps, ensure everything is backed up, as changes will be made to folders as described below.

Reply:
Replaced